qa/workunits/cephtool/test.sh

```diff
@@ -1747,6 +1747,26 @@ function test_mon_osd_pool_quota()
   ceph osd pool get-quota tmp-quota-pool | grep 'max bytes.*10B'
   ceph osd pool get-quota tmp-quota-pool | grep 'max objects.*10M objects'
```

Comment:
the output of  `ceph osd pool get-quota tmp-quota-pool` will be 


```
quotas for pool 'tmp-quota-pool':
  max objects: 10.49M objects
  max bytes  : 10B
```

that's why all "qa/workunits/cephtool/test.sh" tests in http://pulpito.ceph.com/kchai-2018-02-19_09:13:41-rados-wip-kefu-testing-2018-02-19-1111-distro-basic-mira/ failed.
```
2018-02-19T16:54:57.653 INFO:tasks.workunit.client.0.mira101.stderr:/home/ubuntu/cephtest/clone.client.0/qa/workunits/cephtool/test.sh:1747: test_mon_osd_pool_quota:  ceph osd pool get-quota tmp-quota-pool
2018-02-19T16:54:57.656 INFO:tasks.workunit.client.0.mira101.stderr:/home/ubuntu/cephtest/clone.client.0/qa/workunits/cephtool/test.sh:1747: test_mon_osd_pool_quota:  grep 'max objects.*10M objects'
2018-02-19T16:54:58.094 INFO:tasks.workunit.client.0.mira101.stderr:/home/ubuntu/cephtest/clone.client.0/qa/workunits/cephtool/test.sh:1: test_mon_osd_pool_quota:  rm -fr /tmp/cephtool.vxM
```

so, we might want to replace "`'max objects.*10M objects'`" with something like 

```'max objects.*10\(\.[0-9]\+\)\?M objects'```

Reply:
hmm no something is wrong here in set_quota. Probably the base-2 number parser is used for objects. I'll have a look.

Reply:
sorry for the confusion. i didn't claim that set_quota is buggy. we just need to fix the test.

Reply:
No but I'm claiming that (likely) set-quota is buggy. The test sets the quota to 10M objects and the quota is reported as 10.49M

Reply:
ahh, right. that makes sense.

Reply:
yeah set quota uses ```strict_iecstrtoll``` regardless of wether object or byte quotas are set. See [here](https://github.com/jan--f/ceph/blob/60c24b1463453eee1a7627714e98bef164e6e513/src/mon/OSDMonitor.cc#L10907).

Will push a fix.